rollingpin/args.py

```diff
@@ -243,3 +257,28 @@ def construct_canonical_commandline(config, args):
             arg_list.extend(command)
 
     return " ".join(arg_list)
+
+
+def _get_available_profiles(profile_dir):
+    profiles = []
+    for f in os.listdir(profile_dir):
+        if f.endswith(".ini"):
+            profile_name, _ = f.split(".", 1)
+            profiles.append(profile_name)
+
+    return profiles
+
+def make_profile_parser(profile_dir="/etc/rollingpin.d/",
+                        available_profiles=None):
```

Comment:
:no_bell: I don't feel strongly on this, but I generally prefer not to add extra paths through functions just for testing purposes. Mocking `_get_available_profiles` in the tests might be a cleaner way to do this.